### ISSUE TITLE
Does it have landscape support?

### ISSUE THREAD

**Issue by Frexuz** [anvilabs/react-native-image-carousel#6]
Looks awesome. Looking for a image carousel that animates rotation like real native examples.
But I don't see any mention of landscape or orientation, so just wondering before I try implementing it :)

**Comment by binchik** [comment]
Hello @Frexuz, no, it does not support landscape yet :(

**Comment by Frexuz** [comment]
A planned feature? :)

**Comment by binchik** [comment]
Definitely, thanks for bringing it up, I didn't event think of landscape before, I'll try to have it done in a week or two.

**Comment by ghost** [comment]
@binchik any news on this?

**Comment by binchik** [comment]
@eleva sorry Eleva, still no landscape support. Didn't have time for that.
